updates/24/asset_collections_updates.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO asset_collections(id, name, symbol) VALUES (262, "Moonwell", "WELL");
+*
```

Comment:
I'm not sure if the upgraded tokens should be a collection. I don't believe so since they are not the same token